CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 - [#279](https://github.com/thanos-io/kube-thanos/pull/279) Change `hashringConfigmapName` to `hashringConfigMapName` in Receive configuration.
 - [#284](https://github.com/thanos-io/kube-thanos/pull/284) Change Receive `PodDisruptionBudget` api version to `policy/v1`
+- [#293](https://github.com/thanos-io/kube-thanos/pull/293) Upgrade to Thanos v0.30.1
```

Comment:
Let's upgrade to v0.30.2?

Reply:
Suuuure! Going for the v0.30.2 ugprade. 💪

Reply:
@yeya24 done in https://github.com/thanos-io/kube-thanos/pull/293/commits/6bf393641fee0d0dc76bddfce5727a7089694094

Reply:
Hmm seems like the changelog needed to updated. No worries, raised a PR: https://github.com/thanos-io/kube-thanos/pull/294

Reply:
Thx for the follow up with the fix, @saswatamcode 🙇